### ISSUE TITLE
Profiler still does not work with Symfony >= 3.4

### ISSUE THREAD

**Issue by ostrolucky** [Mapudo/guzzle-bundle#17]
```
symfony new mapudo 3.4
cd mapudo
composer require mapudo/guzzle-bundle
// apply default configuration from your README.md and enable bundle in AppKernel
// fire $this->container->get('guzzle.client.test_client')->request('get', 'https://jsonplaceholder.typicode.com/posts/1');
```
When you open profiler,

```
UnexpectedValueException:
The profiler template "GuzzleBundle::debug.html.twig" for data collector "guzzle" does not exist.

  at vendor/symfony/symfony/src/Symfony/Bundle/WebProfilerBundle/Profiler/TemplateManager.php:108
  at Symfony\Bundle\WebProfilerBundle\Profiler\TemplateManager->getNames(object(Profile))
     (vendor/symfony/symfony/src/Symfony/Bundle/WebProfilerBundle/Profiler/TemplateManager.php:54)
  at Symfony\Bundle\WebProfilerBundle\Profiler\TemplateManager->getName(object(Profile), 'request')
     (vendor/symfony/symfony/src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php:114)
  at Symfony\Bundle\WebProfilerBundle\Controller\ProfilerController->panelAction(object(Request), '74e873')
     (vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php:151)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php:68)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php:202)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (web/app_dev.php:29)
  at require('/home/gadelat/Documents/mapudo/web/app_dev.php')
     (vendor/symfony/symfony/src/Symfony/Bundle/WebServerBundle/Resources/router.php:42)

```

**Comment by kobelobster** [comment]
I just tried it out. You're right. On a fresh install of Symfony 3.4 the profiler throws an error. This needs to be fixed ASAP.

Thanks for reporting.

**Comment by kobelobster** [comment]
I identified the problem. The issue seems to be the syntax on how templates are extended/included. For example, we extend the templates using

`{% extends "WebProfilerBundle:Profiler:layout.html.twig" %}`
however, since Symfony 3.4 we should use:
`{% extends "@WebProfiler/Profiler/layout.html.twig" %}`

Changing these lines solves the problem. I created an issue in our ticketing system and we will fix this in the near future..

**Comment by kobelobster** [comment]
@ostrolucky We fixed the issue and released a new version 2.1.1. 

We also tested it on a fresh install of 3.3, 3.4 and an existing 3.4 application. If you want, you can try it out, should work now as expected.

Thanks for reporting this issue.
